mobile/src/Student/screens/Game/PhaseOneBasicGamePlay/index.js

```diff
@@ -1,15 +1,15 @@
 import React, { useState, useEffect, useRef } from "react"
 import {
```

Comment:
nit: Looks like your default tab space is set to 2 and causes almost all the lines here to be shown as changes, same is true for the previous file

Reply:
True, that's because 4 is ridiculous